riscv-configuration-structure-draft.adoc

```diff
@@ -57,96 +33,84 @@ method for a static data structure that can accommodate implementation
-The solution will accommodate hundreds of different types of RISC-V core,
-integrated onto thousands of unique SoCs per year, assembled into tens of
-thousands of unique boards per year.
-
-=== Considerations
-
-Software in early bootup stages might want to parse some of the format.
-This limits the complexity of the parsing process. For reference,
-Table #tab:earlyresources[[tab:earlyresources]] lists the resources
-available to software when the first updateable instruction
-executes in some real-world platforms.
-
-.Resources available to software when the first updateable RISC-V
-instruction executes in some real-world platforms.
-[cols="<,<,<,<",options="header",]
-|=====================================================
-|Platform |RAM |Flash/ROM |Clock Speed
-|HiFive1 |16 kB |8 kB |14.4 MHz
-|HiFive1 Rev B |16 kB |4 MB |14.4 MHz
-|HiFive Unleashed |2 MB L2-LIM |32 MB |33.3 MHz
-|Intel Whitley |stackless code? |32MB |probably 3.7GHz
-|ET-SvcProc |1 MB |128 KB |10 MHz
-|ET-AppProcs |256 KB + cache |0 KB |100 MHz+
-|OpenTitan (small option) |64 KB |512 KB |
-|GigaDevice |32 KB |up to 128 KB |108 MHz
-|=====================================================
-
-=== Use Cases
-
-==== System Firmware
+== Representation of Configuration Structure 
+
+=== Schema of Configuration Structure
+
+The schema of Configuration Structure is the data structure that declares
+the data format which can be used to describe the configuration of core
+and hart features. The schema is typically grouped by the extension
+and flexibly support the next generation of extensions.
+
+To accommodate minimal on-chip descriptions on small systems and larger
+descriptions for other use cases, implementation decisions are divided into
+the following three categories:
+
+1. ((Primary)) information is only discoverable by reading the configuration
+structure, or by running a significant amount of code. Examples: the number of
+hardware triggers supported, ...
+2. ((Secondary)) information is discoverable but not straightforward (e.g. WARL
+register). Examples: whether the F extension is supported, ...
+3. Anything else goes in the ((Complete)) section. Examples: the value of XLEN,
+...
+
+=== Contents of Configuration Structure
+
+The Configuration Structure contains a static description of a hardware
+platform. The static descriptions are the values that made follow the
+schema of Configuration Structure. It describes, in varying levels of detail, the
+implementation decisions made by the hardware designer. The description is
+static and is not affected by the current state of the system.
+
+==== Human-Readable Format
+
+ASN.1 defines a value syntax, but it's not well-supported among open source
+solutions. https://www.itu.int/rec/T-REC-X.697/en[ASN.1
+((JER))] which is a JSON representation of the ASN.1 value that used as
+Human-Readable format for the content of Configuration Structure.
+The Human-Readable format is backward compatible when the new extensions
```

Comment:
```suggestion
The Human-Readable format is backward compatible when new extensions
```

Reply:
Thanks for these comments to make the sentence better :).

Reply:
Resolved in V4